Last-Update: 2014-10-19
Forwarded: no
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: sample.ceph.conf updates:

 * corrected URLs.
 * more options and descriptions.

--- a/src/sample.ceph.conf
+++ b/src/sample.ceph.conf
@@ -31,7 +31,7 @@
 #             ; Example: /var/run/ceph/$cluster-$name.asok
 
 [global]
-### http://ceph.com/docs/master/rados/configuration/general-config-ref/
+### http://ceph.com/docs/giant/rados/configuration/general-config-ref/
 
     ;fsid                       = {UUID}    # use `uuidgen` to generate your own UUID
     ;public network             = 192.168.0.0/24
@@ -51,8 +51,8 @@
     ;max open files             = 131072
 
 
-### http://ceph.com/docs/master/rados/operations/authentication
-### http://ceph.com/docs/master/rados/configuration/auth-config-ref/
+### http://ceph.com/docs/giant/rados/operations/authentication
+### http://ceph.com/docs/giant/rados/configuration/auth-config-ref/
 
     # If enabled, the Ceph Storage Cluster daemons (i.e., ceph-mon, ceph-osd,
     # and ceph-mds) must authenticate with each other.
@@ -90,7 +90,7 @@
     ;keyring                  = /etc/ceph/$cluster.$name.keyring
 
 
-### http://ceph.com/docs/master/rados/configuration/pool-pg-config-ref/
+### http://ceph.com/docs/giant/rados/configuration/pool-pg-config-ref/
 
 
     ## Replication level, number of data copies.
@@ -139,7 +139,7 @@
     ;osd crush chooseleaf type = 1
 
 
-### http://ceph.com/docs/master/rados/troubleshooting/log-and-debug/
+### http://ceph.com/docs/giant/rados/troubleshooting/log-and-debug/
 
     # The location of the logging file for your cluster.
     # Type: String
@@ -154,7 +154,7 @@
     ;log to syslog              = true
 
 
-### http://ceph.com/docs/master/rados/configuration/ms-ref/
+### http://ceph.com/docs/giant/rados/configuration/ms-ref/
 
     # Enable if you want your daemons to bind to IPv6 address instead of
     # IPv4 ones. (Not required if you specify a daemon or cluster IP.)
@@ -167,8 +167,8 @@
 ## You need at least one. You need at least three if you want to
 ## tolerate any node failures. Always create an odd number.
 [mon]
-### http://ceph.com/docs/master/rados/configuration/mon-config-ref/
-### http://ceph.com/docs/master/rados/configuration/mon-osd-interaction/
+### http://ceph.com/docs/giant/rados/configuration/mon-config-ref/
+### http://ceph.com/docs/giant/rados/configuration/mon-osd-interaction/
 
     # The IDs of initial monitors in a cluster during startup.
     # If specified, Ceph requires an odd number of monitors to form an
@@ -216,7 +216,7 @@
     # (Default: 900)
     ;mon osd report timeout          = 300
 
-### http://ceph.com/docs/master/rados/troubleshooting/log-and-debug/
+### http://ceph.com/docs/giant/rados/troubleshooting/log-and-debug/
 
     # logging, for debugging monitor crashes, in order of
     # their likelihood of being helpful :)
@@ -238,6 +238,10 @@
 ;    host                       = gamma
 ;    mon addr                   = 192.168.0.12:6789
 
+    # The maximum number of simultaneous scrub operations for a Ceph OSD Daemon.
+    # Type: 32-bit Int
+    # (Default: 1)
+    ;osd max scrubs               = 2
 
 ##################
 ## Metadata servers
@@ -245,7 +249,7 @@
 # experimental support for running multiple metadata servers. Do not run
 # multiple metadata servers in production.
 [mds]
-### http://ceph.com/docs/master/cephfs/mds-config-ref/
+### http://ceph.com/docs/giant/cephfs/mds-config-ref/
 
     # where the mds keeps it's secret encryption keys
     ;keyring                    = /var/lib/ceph/mds/$name/keyring
@@ -279,7 +283,7 @@
 # You need at least one.  Two or more if you want data to be replicated.
 # Define as many as you like.
 [osd]
-### http://ceph.com/docs/master/rados/configuration/osd-config-ref/
+### http://ceph.com/docs/giant/rados/configuration/osd-config-ref/
 
     # The path to the OSDs data.
     # You must create the directory when deploying Ceph.
@@ -339,7 +343,7 @@
     # (Default: false)
     ;osd check for log corruption = true
 
-### http://ceph.com/docs/master/rados/configuration/journal-ref/
+### http://ceph.com/docs/giant/rados/configuration/journal-ref/
 
     # The size of the journal in megabytes. If this is 0,
     # and the journal is a block device, the entire block device is used.
@@ -365,7 +369,7 @@
     ;debug filestore              = 20
     ;debug journal                = 20
 
-### http://ceph.com/docs/master/rados/configuration/filestore-config-ref/
+### http://ceph.com/docs/giant/rados/configuration/filestore-config-ref/
 
     # The maximum interval in seconds for synchronizing the filestore.
     # Type: Double (optional)
@@ -378,12 +382,6 @@
     # (Default: true)
     ;filestore btrfs snap        = false
 
-    # Enables the filestore flusher.
-    # Type: Boolean
-    # Required: No
-    # (Default: false)
-    ;filestore flusher            = true
-
     # Defines the maximum number of in progress operations the file store
     # accepts before blocking on queuing new operations.
     # Type: Integer
@@ -393,7 +391,7 @@
 
     ## Filestore and OSD settings can be tweak to achieve better performance
 
-### http://ceph.com/docs/master/rados/configuration/filestore-config-ref/#misc
+### http://ceph.com/docs/giant/rados/configuration/filestore-config-ref/#misc
 
     # Min number of files in a subdir before merging into parent NOTE: A negative value means to disable subdir merging
     # Type: Integer
@@ -426,6 +424,7 @@
     # (Default: true)
     ;osd crush update on start    = false
 
+
 ;[osd.0]
 ;    host                         = delta
 
@@ -443,7 +442,7 @@
 ## client settings
 [client]
 
-### http://ceph.com/docs/master/rbd/rbd-config-ref/
+### http://ceph.com/docs/giant/rbd/rbd-config-ref/
 
     # Enable caching for RADOS Block Device (RBD).
     # Type: Boolean
@@ -499,7 +498,7 @@
 ## radosgw client settings
 [client.radosgw.gateway]
 
-### http://ceph.com/docs/master/radosgw/config-ref/
+### http://ceph.com/docs/giant/radosgw/config-ref/
 
     # Sets the location of the data files for Ceph Object Gateway.
     # You must create the directory when deploying Ceph.
